### PR TITLE
Add some Build/Test/Clean scripts for local .NET builds.

### PR DESCRIPTION
**Pull request by jthelin**
- Add Build.cmd script for local command line builds.
    [Includes building of local copy of NuGet package below]
- Add Test.cmd script for local command line automated regression tests.
- Add Clean.cmd scripts to remove any local build binaries.
- ~~Add skeleton SparkCLR.nuspec file for creating a NuGet package for Microsoft.SparkCLR~~

I took a best-guess at what the NuGet package for downstream consumption might look like. It may need some iteration :)

Note: SparkCLR.nuspec has been split out into separate PR #9

### CODE REVIEW

File: csharp/SparkCLR.nuspec [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.SparkCLR</id>
+    <version>1.0.0</version>
+    <authors>Kaarthik Sivashanmugam, Renyi Xiong</authors>
+    <owners>Microsoft</owners>
+    <licenseUrl>https://github.com/Microsoft/SparkCLR/blob/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/Microsoft/SparkCLR</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
```
Comment (skaarthik) [inline csharp/SparkCLR.nuspec]: I will get back to this PR after learning more about this tag in the context of this project

File: csharp/SparkCLR.nuspec [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    <description>C# language binding and extensions to Apache Spark</description>
+    <releaseNotes>See release notes on GitHub https://github.com/Microsoft/SparkCLR/releases</releaseNotes>
+    <copyright>Copyright 2015</copyright>
```
Comment (skaarthik) [inline csharp/SparkCLR.nuspec]: We might just want to reuse the Copyright note in the source files

Comment (jthelin) [inline csharp/SparkCLR.nuspec]: I have pushed an update to this PR branch to amend the <copyright> element to match the copyright header in the code, as @skaarthik suggested.

**Comment by jthelin** [comment]
I split out the SparkCLR.nuspec file into separate PR #9 while the ongoing investigations into what contents should be continue.

This PR does not strictly depend on the .nuspec file now.

**Pull request merged**